pool/cqrs_command/invitation_command.ml

```diff
@@ -60,16 +61,31 @@ end = struct
     ; invited_contacts : Pool_common.Id.t list
     }
 
-  let handle (command : t) system_languages i18n_texts =
+  let handle
+    (command : t)
+    ?(skip_already_invited = false)
```

Comment:
Isn't skipping already invited contacts the default behaviour? 

Is there a use case when a contact gets invited twice? There is already a `resend invitation` function. IMO we should use that to make sure that every contact only has one invitation.